### PR TITLE
Plug photosic functions

### PR DESCRIPTION
**Pull request by jrapin**
## Types of changes

- [ ] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Motivation and Context / Related issue

Follow up on #993 

## Checklist

- [x] The documentation is up-to-date with the changes I made.
- [x] I have read the [**CONTRIBUTING**](https://facebookresearch.github.io/nevergrad/contributing.html) document and completed the CLA (see [**CLA**](https://facebookresearch.github.io/nevergrad/contributing.html#contributor-license-agreement-cla)).
- [x] All tests passed, and additional code has been covered with new tests.

### CODE REVIEW

File: nevergrad/functions/photonics/core.py [hunk 1]
```diff
@@ -53,6 +53,12 @@ def _make_parametrization(
     if name == "bragg":
         shape = (2, dimension // 2)
         bounds = [(2, 3), (30, 180)]
+    elif name == "cf_photosic_realistic":
+        shape = (2, dimension // 2)
+        bounds = [(1, 9), (30, 180)]
+    elif name == "cf_photosic_reference":
+        shape = (1, dimension)
+        bounds = [(30, 180)]
```
Comment (jrapin) [inline nevergrad/functions/photonics/core.py:61]: @Ellawin @AnMoreau what should be the ranges of values for inputs of both functions?

Comment (jrapin) [reply nevergrad/functions/photonics/core.py:61]: also, are they the name you want the functions to be known as?

Comment (Ellawin) [reply nevergrad/functions/photonics/core.py:61]: Yes

Comment (jrapin) [reply nevergrad/functions/photonics/core.py:61]: @Ellawin that answers only one of the two questions :p the other is what should be the range of values for each parts of the vectors. so far `realistic` gets 1 to 9 (that you stated) for permittivity (if I recall correctly). and 30 to 180 for the other part (depth?). For `reference`, it's 30 to 180 as well for the whole vector.

Comment (Ellawin) [reply nevergrad/functions/photonics/core.py:61]: Oh yes sorry, the bounds are perfectly correct !

**Comment by jrapin** [comment]
@teytaud All seems good and confirmed by @Ellawin , I'm merging this, which adds `Photonics` sub-cases.

**Pull request merged**